### PR TITLE
Subalakshmi

### PR DESCRIPTION
**Pull request by subalakshmi25**
function program in js

### CODE REVIEW

File: Subalakshmi/filter.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	var large = arr[0];
+	for(var i=0; i<arr.length; i++){
+		if(large < arr[i]) {
+			large = arr[i];
+		}
+	}
+	return large;
+}
+var input ={name:"suba", notes: [3,4,9],topnotes:''};
+ input.topnotes = largest(input.notes);                   //function call
```
Comment (MurugalakshmiM) [inline Subalakshmi/filter.html:12]: check mutable concepts.

File: Subalakshmi/toparray.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    person.name = name;
+    
+    rl.question('Enter no of elements:',(number)=>{
+        recursiveReadline(number);    //recursive function used
+    });
+});
+
+function recursiveReadline(number){
+    rl.question('Enter elements:',(elements)=>{
+        for(let i=0;i<number;i++){
```
Comment (MurugalakshmiM) [inline Subalakshmi/toparray.html]: check with this loop

File: Subalakshmi/scope.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<body>
+<p>The scope chain in js</p>
+<script>
+var shop= 'shop';
```
Comment (MurugalakshmiM) [inline Subalakshmi/scope.html:6]: use proper variable type according to the version of ECMA script you are using.

Comment (subalakshmi25) [reply Subalakshmi/scope.html:6]: ok ,i will adapt  .. I using let and const variable type in my upcoming programming.

File: Subalakshmi/toparray.html (hunk 2 above)
Comment (subalakshmi25) [reply Subalakshmi/toparray.html]: ok i correct the loop.

**Review by pbravichandhar — APPROVED** [review]

**Pull request merged**